### ISSUE TITLE
Error when trying to "Run As Maven"

### ISSUE THREAD

**Issue by cvgaviao** [ifedorenko/com.ifedorenko.m2e.mavendev#12]
I need to add a feature into a old maven plugin and then when I tried to debug it I received this error in Eclipse 201906:

```
Error: Could not find or load main class com.ifedorenko.m2e.mavendev.junit.runtime.internal.RemoteTestRunner
Caused by: java.lang.ClassNotFoundException: com.ifedorenko.m2e.mavendev.junit.runtime.internal.RemoteTestRunner

```

I have this installed from eclipse repository:
```
  m2e - Maven Integration for Eclipse (includes Incubating components)	1.12.0.20190529-1916	org.eclipse.m2e.feature.feature.group	Eclipse.org - m2e

  Maven Development Tools	0.3.0.201506181201	com.ifedorenko.m2e.mavendev.feature.feature.group	Igor Fedorenko
```
Am I missing something ?

**Comment by cvgaviao** [comment]
Seem that the version on http://ifedorenko.github.com/m2e-extras/ is outdated ?

**Comment by ifedorenko** [comment]
yeah, http://ifedorenko.github.com/m2e-extras/ is outdated, haven't touched it since mid 2015. I should probably take it down to avoid further confusion.

**Comment by cvgaviao** [comment]
> yeah, http://ifedorenko.github.com/m2e-extras/ is outdated, haven't touched it since mid 2015. I should probably take it down to avoid further confusion.

I saw that your current version is 0.5.0-SNAPSHOT, perhaps you could release this new version?

And you know the reason for the error that I've reported ?

**Comment by ifedorenko** [comment]
It's been several years since I looked at Maven/Eclipse integration, don't remember much, sorry. I suggest you build `mavendev` tooling locally and install the local build.
